app.py

```diff
@@ -28,18 +29,55 @@ def archive_processed_file(bucket, key):
     """
 
     global s3_resource
```

Comment:
We should avoid globals if at all possible. Considered bad code in python.

Reply:
+1 on this comment on globals, its bad practice and changing s3_resource like this in the function can make debugging and understanding the code harder in general.

Reply:
Roger that, all globals will be cleared